acceptance/testutils/testutils.go

```diff
@@ -23,7 +23,7 @@ func SkipAcceptanceTest(t *testing.T) {
 // if wd is and empty string it will default to the current working directory
 func RunCommand(cmdString string, wd string) (stdout string, stderr string, exitCode int) {
 	cmds := strings.Split(cmdString, " ")
-	cmd := exec.Command(cmds[0], cmds[1:]...)
+	cmd := exec.Command(cmds[0], cmds[1:]...) //#nosec, nolint:gosec //code only used for testing
```

Comment:
can we configure whichever check that sees this to ignore the test files? otherwise we're going to have lots of these

Reply:
possibly

Reply:
gosec ignore test files by default https://github.com/securego/gosec#excluding-test-files-and-folders
but this is a utility so ive added the folder to an ignore list